src/NuGet.Core/NuGet.Protocol.Core.v3/RemoteRepositories/RemoteV2FindPackageByIdResource.cs

```diff
@@ -126,6 +127,7 @@ private Task<IEnumerable<PackageInfo>> EnsurePackagesAsync(string id, Cancellati
                         // So we decide to leave current logic and observe.
                         using (var data = await _httpSource.GetAsync(
                             uri,
+                            new[] { new MediaTypeWithQualityHeaderValue("application/atom+xml"), new MediaTypeWithQualityHeaderValue("application/xml") },
```

Comment:
nit: this looks like it should be a shared static so since it doesn't change but it used all over and recreated many times